lib/client/client_store.go

```diff
@@ -249,3 +252,11 @@ func LoadKeysToKubeFromStore(profile *profile.Profile, dirPath, teleportCluster,
 	}
 	return kubeCert, privKey, nil
 }
+
+func noCredentialsError(err error) error {
+	return trace.Wrap(err, "no credentials")
+}
+
+func isNoCredentialsError(err error) bool {
+	return strings.Contains(err.Error(), "no credentials")
```

Comment:
Should this line first check if `trace.IsNotFound` to avoid a situation where another type of error uses "no credentials" in its error message?

I'm no expert on this but the Go convention of relying solely or the error message instead of something akin to a class/type/variant always seemed a bit reckless to me. 😅 So I don't really know how this class of problems is usually dealt with in Go.

Reply:
Fair point. I've changed it so `errNoCredentials` is the inner `trace.NotFoundError`. This way, we can use `errors.Is(err, errNoCredentials)`, which is more idiomatic for Go.